File: src/api/dotnet/SeqSort.cs
﻿/*++
Copyright (c) 2016 Microsoft Corporation

Module Name:

    SeqSort.cs

Abstract:

    Z3 Managed API: Sequence Sorts

Author:

    Christoph Wintersteiger (cwinter) 2012-11-23

Notes:
    
--*/

using System.Diagnostics;
using System;

namespace Microsoft.Z3
{
    /// <summary>
    ///  A Sequence sort
    /// </summary>
    public class SeqSort : Sort
    {
        #region Internal
        internal SeqSort(Context ctx, IntPtr obj)
            : base(ctx, obj)
        {
            Debug.Assert(ctx != null);
        }
        #endregion
    }
}

File: src/api/java/SeqSort.java
/**
Copyright (c) 2012-2014 Microsoft Corporation
   
Module Name:

    SeqSort.java

Abstract:

Author:

    @author Christoph Wintersteiger (cwinter) 2012-03-15

Notes:
    
**/ 

package com.microsoft.z3;

/**
 * A Sequence sort
 **/
public class SeqSort<R extends Sort> extends Sort
{
    SeqSort(Context ctx, long obj)
    {
        super(ctx, obj);
    }
}
